lib/vrl/stdlib/src/parse_aws_cloudwatch_log_subscription_message.rs

```diff
@@ -29,16 +28,10 @@ impl Function for ParseAwsCloudWatchLogSubscriptionMessage {
                             "message": "{\"bytes\":26780,\"datetime\":\"14/Sep/2020:11:45:41-0400\",\"host\":\"157.130.216.193\",\"method\":\"PUT\",\"protocol\":\"HTTP/1.0\",\"referer\":\"https://www.principalcross-platform.io/markets/ubiquitous\",\"request\":\"/expedite/convergence\",\"source_type\":\"stdin\",\"status\":301,\"user-identifier\":\"-\"}"
                         }
                     ]
-                }'))
+                }')
             "#},
             result: Ok(
-                // TODO: Remove string literal and use object literal instead.
-                // When removing the string quotes, it doesn't seem to parse, but works fine in the CLI?
-                // I boiled it down to timestamps in arrays failing to parse, e.g
-                //     { "foo": [t'2021-02-02T19:41:00Z'] }
-                // or
-                //     { "foo": [{ "timestamp": 2021-02-02T19:41:00Z }] }
-                r#"s'{"log_events":[{"id":"35683658089614582423604394983260738922885519999578275840","message":"{\"bytes\":26780,\"datetime\":\"14/Sep/2020:11:45:41-0400\",\"host\":\"157.130.216.193\",\"method\":\"PUT\",\"protocol\":\"HTTP/1.0\",\"referer\":\"https://www.principalcross-platform.io/markets/ubiquitous\",\"request\":\"/expedite/convergence\",\"source_type\":\"stdin\",\"status\":301,\"user-identifier\":\"-\"}","timestamp":"2020-09-14T19:09:29.039+00:00"}],"log_group":"test","log_stream":"test","message_type":"DATA_MESSAGE","owner":"111111111111","subscription_filters":["Destination"]}'"#,
+                r#"{"log_events":[{"id":"35683658089614582423604394983260738922885519999578275840","message":"{\"bytes\":26780,\"datetime\":\"14/Sep/2020:11:45:41-0400\",\"host\":\"157.130.216.193\",\"method\":\"PUT\",\"protocol\":\"HTTP/1.0\",\"referer\":\"https://www.principalcross-platform.io/markets/ubiquitous\",\"request\":\"/expedite/convergence\",\"source_type\":\"stdin\",\"status\":301,\"user-identifier\":\"-\"}","timestamp":"2020-09-14T19:09:29.039Z"}],"log_group":"test","log_stream":"test","message_type":"DATA_MESSAGE","owner":"111111111111","subscription_filters":["Destination"]}"#,
```

Comment:
Now that it can be parsed as VRL expression, this would benefit from some newlines with `indoc`.